### PR TITLE
py-neurokit2: add 0.1.4.1

### PR DESCRIPTION
**Pull request by manuelakuhn**

### CODE REVIEW

**Review by adamjstewart — CHANGES_REQUESTED** [review]
It looks like there's a missing build dep:
```python
depends_on('py-pytest-runner', type='build')
```

**Comment by manuelakuhn** [comment]
Just checked again and installing with `spack install --test=root py-neurokit2`, loading and running the [quick example](https://github.com/neuropsychology/NeuroKit/tree/v0.1.4.1#quick-example) all work without this dependency. Should I add it, nevertheless?

**Comment by adamjstewart** [comment]
Yeah, you should still add it. If it's not already installed, setuptools will automatically download and install it, which won't work on air-gapped networks.

File: var/spack/repos/builtin/packages/py-neurokit2/package.py [hunk 1]
```diff
@@ -19,9 +19,11 @@ class PyNeurokit2(PythonPackage):
     homepage = "https://github.com/neuropsychology/NeuroKit"
     pypi     = "neurokit2/neurokit2-0.1.2.tar.gz"
 
+    version('0.1.4.1', sha256='226bb04bb369d8bb87d99831f0a93cd8d0ed96fdc500f63de0b3550082876f6e')
     version('0.1.2', sha256='5ef40037c2d7078ecb713ab0b77b850267babf133856b59595de9613f29787bc')
 
     depends_on('py-setuptools@040.6.0:', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-neurokit2/package.py]:
```suggestion
    depends_on('py-setuptools@40.6.0:', type='build')
```
Just noticed this. I don't think it makes a difference, but might as well remove the typo

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-neurokit2/package.py]: Oh that's easy to overlook. Thanks!

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**